chrome/content/zotero-better-bibtex/cayw.coffee

```diff
@@ -182,11 +182,12 @@ Zotero.BetterBibTeX.CAYW.Formatter = {
   pandoc: (citations) ->
     formatted = []
     for citation in citations
+      locator = if citation.locator then "#{Zotero.BetterBibTeX.CAYW.shortLocator[citation.label]} #{citation.locator}" else ''
       cite = ''
       cite += "#{citation.prefix} " if citation.prefix
       cite += '-' if citation['suppress-author']
       cite += "@#{citation.citekey}"
-      cite += ", #{citation.label} #{citation.locator}" if citation.locator
+      cite += ", #{locator}"
```

Comment:
This always inserts the comma, even if the locator isn't set. Is this desired? I don't know pandoc well enough to say.